insights/mappers/systemctl_show.py

```diff
@@ -1,11 +1,62 @@
+"""
+/bin/systemctl show <Services_Name> - Command
+=============================================
+
+This command shows all the paramters for this service, these parameters can be set
+through unit files corresponding to that service or from command line.
+
+Typical output of ``/bin/systemctl show mariadb`` command is::
+
+    Type=simple
+    Restart=no
+    NotifyAccess=none
+    RestartUSec=100ms
+    TimeoutStopUSec=5min
+    ExecStartPre={ path=/usr/libexec/mariadb-prepare-db-dir ; argv[]=/usr/libexec/mariadb-prepare-db-dir %n ; ignore_errors=no ; start_time=[Mon 2017-05-22 06:49:01 EDT] ; stop_time=[Mon 2017-05-22 06:49:02 EDT] ; pid=1946 ; code=exited ; status=0 }
+    ExecStart={ path=/usr/bin/mysqld_safe ; argv[]=/usr/bin/mysqld_safe --basedir=/usr ; ignore_errors=no ; start_time=[Mon 2017-05-22 06:49:02 EDT] ; stop_time=[n/a] ; pid=2304 ; code=(null) ; status=0/0 }
+    ExecStartPost={ path=/usr/libexec/mariadb-wait-ready ; argv[]=/usr/libexec/mariadb-wait-ready $MAINPID ; ignore_errors=no ; start_time=[Mon 2017-05-22 06:49:02 EDT] ; stop_time=[Mon 2017-05-22 06:49:12 EDT] ; pid=2305 ; code=exited ; status=0 }
+    Slice=system.slice
+    ControlGroup=/system.slice/mariadb.service
+    After=network.target -.mount systemd-journald.socket tmp.mount basic.target syslog.target system.slice
+    MemoryCurrent=18446744073709551615
+    ...
+
+Examples:
+    >>> SrvMariaDB = shared[SystemctlShowMariaDB]
+    >>> SrvMariaDB["After"]
+    network.target -.mount systemd-journald.socket tmp.mount basic.target syslog.target system.slice
+
+This command show the all the paramters for this service, these parameters can be set
+through unit files corresponding to that service or from command line.
+
+Typical output of ``/bin/systemctl show openstack-cinder-volume`` command is::
+
+    Restart=no
+    NotifyAccess=none
+    RestartUSec=100ms
+    TimeoutStartUSec=1min 30s
+    TimeoutStopUSec=1min 30s
+    WatchdogUSec=0
+    LimitCORE=18446744073709551615
+    LimitRSS=18446744073709551615
+    LimitNOFILE=65536
+    LimitAS=18446744073709551615
+    LimitNPROC=63391
+    Transient=no
+    ...
+
+Examples:
+    >>> data = shared[SystemctlShowCinderVolume]
+    >>> data["LimitNOFILE"]
+    65536
+"""
 from .. import Mapper
 from .. import mapper
-from .. import get_active_lines
-
+from .. import LegacyItemAccess
+from ..mappers import split_kv_pairs
 
-@mapper('systemctl_cinder-volume')
-class SystemctlShowCinderVolume(Mapper):
 
+class SystemctlShow(LegacyItemAccess, Mapper):
```

Comment:
@vishwanathjadhav this class needs to be documented since the mapper classes are based on this class.  Take a look at the generated documentation and you'll see that the base class `SystemctlShow` but there is no documentation for this class.